Dockerfile

```diff
@@ -10,24 +10,37 @@ LABEL name="kafka" version=${KAFKA_VERSION}
 LABEL org.opencontainers.image.source https://github.com/kubedb/kafka-docker
 
 
-# https://archive.apache.org/dist/kafka contains all the kafka version binary
+# ref: https://archive.apache.org/dist/kafka contains all the kafka version binary
+# ref: https://github.com/edenhill/kcat used for installing kafkacat cli
 RUN apt-get update \
- && apt-get install wget \
+ && apt-get install -y wget sudo \
+ && apt-get -y install kafkacat \
```

Comment:
change it to - `apt-get install -y wget sudo kafkacat`
also add a small comment to explain why we are installing sudo